subsys/bluetooth/services/Kconfig.dis

```diff
@@ -0,0 +1,109 @@
+# Kconfig - Bluetooth GATT Device Information service
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_GATT_DIS
+	bool "Enable BLE GATT Device Information service"
+	default n
+
+if (BT_GATT_DIS)
+
+choice
+	prompt "Device Information service logging level"
+	default BT_GATT_DIS_LOG_LEVEL_INF
+
+ config BT_GATT_DIS_LOG_LEVEL_OFF
+	bool "Off"
+config BT_GATT_DIS_LOG_LEVEL_ERR
+	bool "Error"
+config BT_GATT_DIS_LOG_LEVEL_WRN
+	bool "Warning"
+config BT_GATT_DIS_LOG_LEVEL_INF
+	bool "Info"
+config BT_GATT_DIS_LOG_LEVEL_DBG
+	bool "Debug"
+
+ endchoice
+
+ config BT_GATT_DIS_LOG_LEVEL
+	int
+	default 0 if BT_GATT_DIS_LOG_LEVEL_OFF
+	default 1 if BT_GATT_DIS_LOG_LEVEL_ERR
+	default 2 if BT_GATT_DIS_LOG_LEVEL_WRN
+	default 3 if BT_GATT_DIS_LOG_LEVEL_INF
+	default 4 if BT_GATT_DIS_LOG_LEVEL_DBG
+
+
+config BT_GATT_DIS_MODEL
+	string "Model name"
+	default SOC
+	help
+	 The device model inside Device Information Service.
+
+config BT_GATT_DIS_MANUF
+	string "Manufacturer name"
+	default "Manufacturer"
+	help
+	 The device manufacturer inside Device Information Service.
+
+config BT_GATT_DIS_PNP
+	bool "Enable PnP_ID characteristic"
+	default n
+	help
+	 Enable PnP_ID characteristic in Device Information Service.
+
+if (BT_GATT_DIS_PNP)
+
+config BT_GATT_DIS_PNP_VID_SRC
+	int "Vendor ID source"
+	range 1 2
+	default 1
+	help
+	 The Vendor ID Source field designates which organization assigned the value used in the Vendor ID field value.
+	 The possible values are:
+	 - 1 Bluetooth SIG, the Vendor ID was assigned by the Bluetooth SIG
+	 - 2 USB IF, the Vendor ID was assigned by the USB IF
+
+config BT_GATT_DIS_PNP_VID
+	hex "Vendor ID"
+	range 0 0xFFFF
+	default 0
```

Comment:
I wonder if we wouldn't happen to have a default as zephyr, afaik we did have something for USB which perhaps we can use here as well.

Reply:
Don't know myself, I guess it could be changed later?